metered/src/common/throughput/tx_per_sec.rs

```diff
@@ -30,8 +32,14 @@ impl<T: Instant> RecordThroughput for std::cell::RefCell<TxPerSec<T>> {
     }
 }
 
+impl<T: Instant> Clear for std::cell::RefCell<TxPerSec<T>> {
+    fn clear(&self) {
+        self.borrow_mut().clear();
+    }
+}
+
 impl<T: Instant> TxPerSec<T> {
-    pub fn on_result(&mut self) {
+    pub(crate) fn on_result(&mut self) {
```

Comment:
Any reason you're restricting the visibility here?

Reply:
Leaving these methods public would require that they be documented. But these methods don't appear to serve any public purpose. They can't be used to implement `Clear` or `RecordThroughput` on `TxPerSec<T>`, as those traits require the method receivers to be immutable.  Thus, they are merely an internal implementation detail, used when wrapping `TxPerSec` with `RefCell`.

I suppose they could be left public and documented, in case someone wants to wrap `TxPerSec` on something other than `RefCell` using the newtype pattern to get around the orphan rule.

Reply:
Alright, sounds fair to me.